roboflow/roboflowpy.py

```diff
@@ -16,7 +16,10 @@
 
 
 def login(args):
-    roboflow.login()
+    if args.force:
+        roboflow.login(force=True)
+    else:
+        roboflow.login()
```

Comment:
I guess you can trust argparse here.

```suggestion
    roboflow.login(force=args.force)
```